themes.gitconfig

```diff
@@ -384,3 +384,42 @@
     merge-conflict-theirs-diff-header-style = "#FABD2F" bold overline
     merge-conflict-ours-diff-header-decoration-style = ''
     merge-conflict-theirs-diff-header-decoration-style = ''
+
+[delta "discord-dark"]
+    # author: https://github.com/ILikePlayingGames
+    dark                          = true
+    commit-style                  = omit
+    file-style                    = 34 ul
+    file-decoration-style         =
+    hunk-header-style             = omit
+    minus-style                   = 31
+    minus-non-emph-style          = 31
+    minus-emph-style              = 40
+    minus-empty-line-marker-style = 31
+    zero-style                    = 30
+    plus-style                    = 32
+    plus-non-emph-style           = 32
+    plus-emph-style               = 40
+    plus-empty-line-marker-style  =
+    grep-file-style               = 34
+    grep-line-number-style        = 34
+    whitespace-error-style        = 41
+    blame-code-style              = omit
+    true-color                    = never
+    file-added-label              = added:
+    file-modified-label           = changed:
+    file-removed-label            = deleted:
+    file-renamed-label            = renamed:
+    right-arrow                   = >
+    hyperlinks                    = false
+    inspect-raw-lines             = true
+    keep-plus-minus-markers       = true
+    line-numbers                  = false
+    line-fill-method              = ansi
+    navigate                      = false
+    navigate-regex                =
```

Comment:
It feels like this blank entry should not be necessary -- does it work without it?